website/addons/wiki/shareServer.js

```diff
@@ -60,31 +60,16 @@ app.use(morgan('common'));
 
 // Allow CORS
 app.use(function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', settings.corsAllowOrigin);
+    res.header('Access-Control-Allow-Origin', 'http://localhost:5000');
```

Comment:
This needs to be configurable for non-local environments, also https://github.com/CenterForOpenScience/osf.io/pull/6681/files#diff-1ccc7cbdcef56c872e6db4c76737d8fcR19 indicates it is defaulted to this value.